src/editor/scheduler-editor-time.ts

```diff
@@ -395,11 +401,15 @@ export class SchedulerEditorTime extends LitElement {
     return actions.map(action => {
       return Object.entries(this.actions!.find(e => compareActions(e, action, true))!.variables!).map(
         ([field, variable]) => {
+          let header: string = variable.name || PrettyPrintName(field);
+          if (header.toLowerCase() === 'time') {
+            const entity = this.entities?.[0];
+            if (entity) header += ` (${PrettyPrintName(this.getEntityName(entity))})`;
+          }

```

Comment:
This `header` logic changes the section title from "TIME" to "TIME (&lt;entity name&gt;)" to help disambiguate between the two time sections — see red arrows in the screenshots below. ***This only applies when the selected entity is a time entity.*** If the user selected a different entity, say switch, the section title remains as before (not including the entity name).

_Without the `header` logic:_
<img width="70%" alt="New Schedule - Time Editor - ambiguous highlight" src="https://github.com/nielsfaber/scheduler-card/assets/15091591/62c6054f-650f-4128-952b-209730b58775" />

_With the `header` logic:_
<img width="70%" alt="New Schedule - Time Editor - highlight" src="https://github.com/nielsfaber/scheduler-card/assets/15091591/d852f07a-063c-466a-aabd-9341d617e4c8" />